index.js

```diff
@@ -5,6 +5,7 @@ const commonDir = require('commondir');
 const pkgDir = require('pkg-dir');
 const makeDir = require('make-dir');
 
+const MODULES_DIRECTORIES_NAMES = ['node_modules', '.yarn'];
```

Comment:
Shouldn't it be?

```suggestion
const MODULE_DIRECTORY_NAMES = ['node_modules', '.yarn'];
```

Same with the other function and variable names.

Reply:
I agree, this should not be plural.